packages/web/src/lib/utils.ts

```diff
@@ -27,3 +27,18 @@ export function getTitleForSession(session: SessionType) {
   const titleCell = session.cells.find((cell: CellType) => cell.type === 'title') as TitleCellType;
   return titleCell?.text;
 }
+
+// POST feedback to Google Sheets
+// Because of CORS, we don't get a response here so this is a fire and forget.
```

Comment:
Why not make this request from the API where CORS is not an issue?

Reply:
Duh, I was clearly tired as I did not think of this. Fixed.